core/App/ehsm_napi.cpp

```diff
@@ -1562,21 +1565,21 @@ char* NAPI_VerifyQuote(const char *quote_base64, const char *nonce)
     ehsm_status_t ret = EH_OK;
     sgx_ql_qv_result_t verifyresult;
     bool result = false;
-
     ehsm_data_t quote;
+    // string mrenclave_base = base64_decode(mrenclave_req_base64);
+    // string mrsigner_base = base64_decode(mrsigner_req_base64);
+    // printf("mrenclave_base: %s\n",mrenclave_base.c_str());
+    // printf("mrsigner_base: %s\n",mrsigner_base.c_str());
     memset(&quote, 0, sizeof(quote));
-
     string quote_str = base64_decode(quote_base64);
     quote.datalen = quote_str.size();
     quote.data = (uint8_t*)quote_str.data();
-
     if(quote.datalen == 0 || quote.datalen > EH_QUOTE_MAX_SIZE){
         retJsonObj.setCode(retJsonObj.CODE_BAD_REQUEST);
         retJsonObj.setMessage("The quote's length is invalid.");
         goto out;
     }
-
-    ret = VerifyQuote(&quote, &verifyresult);
+    ret = VerifyQuote(&quote, mrsigner_req_base64, mrenclave_req_base64, &verifyresult);
```

Comment:
mrsigner_req is not base64

Reply:
if you passed them by base64, why not decode them in this function?

Reply:
Copy understood